### ISSUE TITLE
sparse padding flag ignored for more than 20 columns

### ISSUE THREAD

**Issue by camroberts** [paulfitz/daff#125]
Firstly, thanks for the great library.  I'm finding it very useful, but believe I've found an issue.

For csv files with more than 20 columns the sparse aligned column option does not seem to work.  The output is always dense. 

Steps to reproduce compare a csv file with more than 20 columns to another:
```
daff left-more-than-20.csv right-more-than-20.csv
daff --padding smart left-more-than-20.csv right-more-than-20.csv
daff --padding dense left-more-than-20.csv right-more-than-20.csv
daff --padding sparse left-more-than-20.csv right-more-than-20.csv
```

All these result in the same dense output.

While I can understand that the "smart" option would show dense output for files with higher column counts, I would have expected that forcing "sparse" would align columns regardless of the number.

**Comment by paulfitz** [comment]
Hmm yes I see what you're saying.  Working on a fix in #126.  Thanks for reporting.

**Comment by paulfitz** [comment]
I believe this is fixed as of 1.3.38.  Please reopen if not.  Cheers!
